vignettes/calculate_known_outcomes.Rmd

```diff
@@ -0,0 +1,299 @@
+total number of deaths and the total number of cases or hospitalisations (respectively) — would be accurate. These basic estimates are referred 
+to as naive estimators. However, due to delays between onset or hospitalisation and death for all known diseases, this synchronisation is
+never true.
+
+For example, suppose 10 people start showing symptoms of a specific disease on a given day and the end of that day all remain alive. Suppose
+that for the next 5 days, the numbers of new cases continue to rise until they reach 100 new cases on day 5. However, suppose that by day 5, 
+all infected individuals remain alive. The naive estimate of the CFR would be zero calculated at the end of the first 5 days would be zero,
+because there would have been zero deaths in total. Even after deaths begin to occur, this lag between the ascertainment of a case or 
+hospitalisation and outcome leads to a consistently biased estimate. Hence, adjusting for such delays using an appropriate delay distribution 
+is essential for accurate estimates of severity.
+
+# Data required
+
+This method require three sets of data:
+
+* A time-series of cases;
+* A time-series of deaths;
+* A delay distribution, describing the probability an individual will die $t$ days after they were initially exposed. Such distributions come from the literature, where studies have typically fit distributions to data describing the process. 
+
+In practice, the time-series of cases and deaths will already be together, given that they usually originate from sources that will have typically collated them into a single data file.
+
+# Example with data from the 1976 Ebola outbreak in the Democratic Republic of the Congo
+
+We use data from the 1976 Ebola outbreak. First of all, we plot the data from the entire outbreak. Then we subset 
+the data isolating about half of it, to emulate estimating severity during an ongoing outbreak.
+
+## Plotting the raw data
+
+First of all, we plot the raw data
+
+```{r message = FALSE, warning = FALSE, eval = TRUE, fig.cap = "\\label{fig:raw_data_ebola} **Figure 1:** Incidence of cases and deaths over time, taken from the 1976 Ebola outbreak in the Democratic Republic of the Congo. Data retrieved from within the *datadelay* package"}
+data("ebola1976")
+plot_raw_data(ebola1976, together = TRUE)
+```
+
+The argument `together = FALSE` of the `plot_raw_data()` function plots the case and death times series on a 
+single figure, rather than two separate figures. This is useful if the raw values are similar and they can 
+effectively share a scale. Otherwise, using the `together = TRUE` plot them on separate figures. The default 
+argument is `together = TRUE`.
+
+We will focus on the roughly the first half of this dataset. We do so by subsetting the data so that we only include
+days before the 30th September 1976, using the following command:
+
+```{r, message = FALSE, warning = FALSE, eval = TRUE}
+df_ebola_subset <- subset(ebola1976, date <= "1976-09-30")
```

Comment:
```suggestion
df_ebola_subset <- subset(ebola1976, date <= "1976-09-18")
```

Reply:
Suggest using `lubridate::ymd("1976-09-18")` or base R's as.Date(...) to ensure you're using a date format. Actually, before saving the data in './data/', ensure all the columns have the right format.